### PR TITLE
libfabric: fix test not doing anything

### PR DESCRIPTION
**Pull request by Bo98**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Was being interpreted as a shell comment due to a typo so nothing was being run.

### CODE REVIEW

File: Formula/libfabric.rb [hunk 1]
```diff
@@ -26,6 +26,6 @@ def install
   end
 
   test do
-    system "#(bin}/fi_info"
+    system "#{bin}/fi_info"
```
Comment (SMillerDev) [inline Formula/libfabric.rb:29]: Can we also check some output while we're at it?

Comment (Bo98) [reply Formula/libfabric.rb:29]: I know very little about the software and don't know what most of the output is. What does it output on your system? I could do something like searching for `provider: sockets` as I imagine every system has a socket interface.

Comment (SMillerDev) [reply Formula/libfabric.rb:29]: Seems good to me, I have plenty of mentions of those too.

**Comment by Bo98** [comment]
`adios2` failure is known and has been reported.

**Pull request merged**